improver/utilities/threshold_interpolation.py

```diff
@@ -193,15 +181,15 @@ def process(
         6. Applies the original mask to the new cube if it exists.
 
         Args:
-            forecast_at_thresholds (Cube):
+            forecast_at_thresholds:
                 Cube expected to contain a threshold coordinate.
 
         Returns:
             Cube:
                 Cube with forecast values at the desired set of thresholds.
                 The threshold coordinate is always the zeroth dimension.
         """
-        self.threshold_coord = find_threshold_coordinate(forecast_at_thresholds).name()
+        self.threshold_coord = find_threshold_coordinate(forecast_at_thresholds)
```

Comment:
Line 196/197 below:
Code coverage is suggesting you don't have test coverage of this. You'll need a unit test with a cube that has a realization dimension.